rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1694,12 +1696,13 @@ message ManageNeuron {
   // neuron's tokens (if no argument is provided) to a given ledger account.
   // Thereby, the neuron's accumulated fees are burned and (if relevant in
   // the given nervous system) the token equivalent of the neuron's accumulated
-  // maturity are minted and also transferred to the specified account.
+  // maturity is minted and also transferred to the specified account.
   message Disburse {
     message Amount {
       uint64 e8s = 1;
     }
 
+    // (Q: where is the optional?)
```

Comment:
In the protobuf version we're using, all message-typed fields are automatically optional. So `Amount amount = 1;` is the same as `optional Amount amount = 1;`